app/client/screens/Add/Workout.tsx

```
@@ -0,0 +1,483 @@
+  TextInput,
+  TouchableOpacity,
+  Alert,
+} from "react-native";
+import { Title, Button } from "@/client/components";
+import { AntDesign } from "@expo/vector-icons";
+import { AlertDialog as DiscardWorkoutDialog } from "@/client/components/Dialogs";
+import { useNavigation } from "@react-navigation/native";
+import { NativeStackNavigationProp } from "@react-navigation/native-stack";
+import { RootStackParamList } from "@/types/AppModels";
+import { useWorkout } from "@/client/contexts/WorkoutContext";
+import { Exercise, Set } from "@/types/DatabaseModels";
```

Biome (1.9.4)

[error] Do not shadow the global "Set" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)